editor/src/components/text-editor/text-editor.tsx

```diff
@@ -158,15 +169,69 @@ export const TextEditorWrapper = React.memo((props: TextEditorProps) => {
   return React.createElement(component, passthroughProps, <span {...editorProps} />)
 })
 
-function setSelectionToEnd(element: HTMLSpanElement) {
-  const range = document.createRange()
-  range.selectNodeContents(element)
-  range.collapse(false)
+async function setSelectionToOffset(
+  element: HTMLSpanElement,
+  scale: number,
+  cursorPosition: Coordinates | null,
+) {
+  const sel = window.getSelection()
```

Comment:
Minor, but why us `sel` and not `selection`?

Reply:
https://github.com/concrete-utopia/utopia/pull/3061/commits/d0ea2b8d246bce78bacc55825e24fc5f4340632d